src/components/Recipes/RecipePage.tsx

```diff
@@ -26,7 +30,9 @@ const RecipePage: React.FC = () => {
         <img src={recipeInfo.image} alt={recipeInfo.title}/>
       </figure>
       <h1>{recipeInfo.title}</h1>
-      <p>{recipeInfo.instructions}</p>
+      <div className="instructions">
```

Comment:
Should be paragraph if it’s a paragraph

Reply:
It's being changed from a paragraph into a list so that the instructions are easier to parse. They are otherwise lumped together unpleasantly.